uaclient/jobs/license_check.py

```diff
@@ -4,39 +4,38 @@
 """
 import logging
 
-from uaclient import config, exceptions
+from uaclient import config, exceptions, jobs, util
 from uaclient.cli import action_auto_attach, setup_logging
 from uaclient.clouds.gcp import GCP_LICENSES, UAAutoAttachGCPInstance
 from uaclient.clouds.identity import get_cloud_type
+from uaclient.config import UAConfig
 from uaclient.util import get_platform_info
 
-LOG = logging.getLogger(__name__)
+LOG = logging.getLogger("ua_lib.license_check.jobs.license_check")
```

Comment:
This is a working approach for the logging. I thought about this for the other timer, and gave up because it would introduce inconsistent behavior for the other jobs. 
Now that we have a dedicated timer, picking up its logger for the job makes more sense. I think this is a good approach.

Reply:
Note to self: The only unnecessary 5-min repeating logs we are suppressing now is from the `uaclient.utils.readurl` method, while checking for the license.

Reply:
Ack, looks good here for scoping the log handler definitions.